server/appengine/src/main/java/who/PublicWhoServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package who;
+
+// All responses are Client-independent.
+// No Client instance is available in this service.
+public class PublicWhoServiceImpl implements PublicWhoService {
+
+  @Override public GetGlobalStatsResponse getGlobalStats(Void request) {
+    // TODO: Configure cache response headers. RFC 2616 Section 9.5 says we can cache
+    // POST requests.  Or we can enable GET method in present.rpc.RpcFilter.
```

Comment:
I think it makes more sense (from an API perspective) to make this a GET method.

Reply:
It does, but RpcFilter forced POST right now.